src/phases/command-phase.ts

```diff
@@ -392,8 +391,7 @@ export class CommandPhase extends FieldPhase {
 
   public cancel(): void {
     if (this.fieldIndex) {
-      globalScene.phaseManager.unshiftPhase(new CommandPhase(0));
-      globalScene.phaseManager.unshiftPhase(new CommandPhase(1));
+      globalScene.phaseManager.pushPhase(new CommandPhase(0), new CommandPhase(1));
```

Comment:
Oh I missed this the first time.
```suggestion
      globalScene.phaseManager.unshiftPhase(new CommandPhase(0), new CommandPhase(1));
```